ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/deneb/helpers/MiscHelpersDeneb.java

```diff
@@ -164,6 +164,11 @@ public int getBlobSidecarsCount(final Optional<SignedBeaconBlock> signedBeaconBl
         .orElse(0);
   }
 
+  public UInt64 computeDenebStartSlot() {
```

Comment:
I think we should use a feature not a milestone in the name. Something like `computeFirstSlotWithBlobSupport`